portfolio/src/main/webapp/script.js

```diff
@@ -59,6 +59,35 @@ function displayCountry() {
     responsePromise.then(handleResponse);
 }
 
+function handleResponse(response) {
+  console.log('Handling the country response.');
+
+  // response.text() returns a Promise, because the response is a stream of
+  // content and not a simple variable.
+  const textPromise1 = response.text();
+  console.log(textPromise1);
+//   const textPromise2 = response[b].text(); 
+//   console.log(textPromise2);
+
+  textPromise1.then(addCountry);
+}
+
+function addCountry(country) {
+  console.log('Country added: ' + country);
+
+  const countryName = document.getElementById('country-name');
+  countryName.innerText = country;
+}
+
+function displayComments() {
+    const title = document.getElementById('content-title');
+    title.innerHTML = "Do you have any feedback on my website? Please feel free to leave a comment.";
+    $(function(){
```

Comment:
I just noticed something on this (And the other three) `load` calls.

The way that this is written actually splits into two parts:
 - `$(someFunction)` is a jQuery shortcut for registering this function/callback to be called when the page has finished loading. This is mainly used by jQuery plugins when the javascript file may be loaded/run before all of the HTML has finished loading
 - In the callback, `$("selector").load(...)` actually triggers the HTTP request and content insertion.

There's no need to have the outer function here at all -- by the time any of the buttons are clicked, the page will have fully loaded already. You can remove this (i.e. lines #85 and #87) entirely with no change to functionality.